serverless.yaml

```diff
@@ -3,7 +3,7 @@ provider:
   name: aws
   stackName: ${{self:service}}-${{self:provider.stage}}
   runtime: nodejs12.x
-  memorySize: 1024
+  memorySize: 512
```

Comment:
Great call out! Will save us some good money!

Reply:
In my experience, 512 is usually a good compromise between cost and performance. 
For the future, there is [a tool](https://www.npmjs.com/package/lumigo-cli#lumigo-cli-analyze-lambda-cost) that analyses cost and tells you the best match for each individual function